DockerHubBackend/DockerHubBackend/Controllers/AuthController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DockerHubBackend.Dto.Request;
+using DockerHubBackend.Repository.Interface;
+using DockerHubBackend.Security;
+using DockerHubBackend.Services.Interface;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.IdentityModel.Tokens.Jwt;
+
+namespace DockerHubBackend.Controllers
+{
+    [Route("api/auth")]
+    [ApiController]
+    public class AuthController : ControllerBase
+    {        
+        private readonly IAuthService _authService;
+        private readonly IConfiguration _configuration;
+
+        public AuthController(IAuthService authService, IConfiguration configuration)
+        {
+            _authService = authService;
+            _configuration = configuration;
+        }
+
+        [HttpPost("login")]
+        public async Task<IActionResult> Login([FromBody] LoginCredentialsDto credentials)
+        {
+            var response = await _authService.Login(credentials);
+            var tokenExpiration = Convert.ToInt32(_configuration["JWT:Expiration"]);
+            Response.Cookies.Append("AuthToken", response.Jwt, new CookieOptions
```

Comment:
Is JWT passed using cookies? Did you consider passing it as a Authorization header?

Reply:
I considered that option, but cookie with HttpOnly, Secure and SameSite properties set is much more secure.